lib/v8-log-to-ticks.js

```diff
@@ -122,7 +122,10 @@ function v8LogToTicks (isolateLogPath, node) {
     const tickStream = parse('ticks.*', (tick) => {
       const addr = tick.s.filter((n, i) => i % 2 === 0)
       var stack = addr.map((n) => codes[n]).filter(Boolean)
-      ticks.push(stack.reverse())
+      const delayMs = options.collectDelay * 1000
+      if (tick.tm > delayMs) {
```

Comment:
Can you try the comparison relative to the first tick's time here like we discussed in slack a while ago?

Reply:
I'll have a look at it!